ci-operator/config/openshift/os/openshift-os-master.yaml

```diff
@@ -53,6 +64,57 @@ tests:
   container:
     from: build-test-qemu-img
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: periodic-os-build
+  cluster: build02
+  cron: '@daily'
+  steps:
+    test:
+    - as: oc-login
+      cli: latest
+      commands: |
+        #!/bin/bash
+        set -euo pipefail
+        # To inject the oc CLI, ci-operator builds a new container from the
+        # provided container dependency (from:) and pushes it to the namespace
+        # Imagestream before creating a new pod to run the commands in. Because
+        # of the size of the coreos-assembler image, this adds quite some time
+        # to the process. So lets use a much smaller Fedora base image.
+        oc login https://api.ci.l2s4.p1.openshiftapps.com:6443 --token="$(cat /service-account-token/image-pusher-service-account-token)"
+        oc registry login --registry=registry.ci.openshift.org --to="$SHARED_DIR/dockercfg.json"
+      credentials:
+      - mount_path: /service-account-token
+        name: rhcos-devel-image-pusher-service-account-token
+        namespace: test-credentials
+      from: fedora
+      resources:
+        limits:
+          devices.kubevirt.io/kvm: "1"
+        requests:
+          cpu: 1000m
+          devices.kubevirt.io/kvm: "1"
+          memory: 3Gi
+    - as: build-test-and-push-img
+      commands: |
+        #!/bin/bash
+        set -euo pipefail
+
+        /src/ci/build-test-qemu.sh
+
+        export REGISTRY_AUTH_FILE="$SHARED_DIR/dockercfg.json"
+        # Tags with the cosa build ID / arch
+        cosa push-container registry.ci.openshift.org/rhcos-devel/rhel-coreos
```

Comment:
OK right, this is failing because it's not being run in the cosa workdir, see this code:
https://github.com/openshift/os/blob/da07aad841be5361fa885620031c6eeeecd6bf38/ci/build-test-qemu.sh#L8

One option is to pass the workdir as a parameter to that test.  Or in a container we can just use a predictable workdir like `/tmp/cosa` or whatever.